nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -171,10 +171,33 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 	pixel_x = 38
 	anchored = TRUE
 	density = FALSE
+	var/place_landing_waypoint = TRUE
 	var/obj/structure/overmap/fighter/mag_locked = null
 	var/obj/structure/overmap/linked = null
 	var/ready = TRUE
 
+/obj/structure/fighter_launcher/launch_only //If you don't want them to also land here.
+	place_landing_waypoint = FALSE
+
+/obj/structure/fighter_launcher/galactica //If it shouldn't actually launch people. But should just catch them.

```

Comment:
This typepath is bad and non-generic. This should be `fighter_launcher/landing` or something to that effect

Reply:
As this appears to be simply a visual bug correction type, the offsets should be configurable at the map level.